apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -454,6 +454,10 @@ private void processAsync(
           } else if (rifRecordEvent.getRecordAction().equals(RecordAction.UPDATE)) {
             loadAction = LoadAction.UPDATED;
             tweakIfBeneficiary(entityManager, loadedBatchBuilder, rifRecordEvent);
+            /*
+             * TODO should we be explicitly blowing up here if we try to UPDATE a not-pre-existing
```

Comment:
hmmm....seems a bit like juggling chainsaws...if we let it slide, and a claim comes in then we risk the RifLoader crapping out. Biut then that begs the question, why would we get an claim for a bene_id that was never delivered with an INSERT?